Gemfile

```diff
@@ -26,7 +26,8 @@ gem 'coffee-rails', '~> 4.2'
 # See https://github.com/rails/execjs#readme for more supported runtimes
 # gem 'therubyracer', platforms: :ruby
 gem 'paperclip'
-gem 'aws-sdk'
+# gem 'aws-sdk', '< 2.0'
```

Comment:
officially supported gem seems to be aws-sdk, '<~ 2'

https://github.com/aws/aws-sdk-ruby